Update FAQ (#875)
cyberbotics/webots: docs/guide/general-faq.md

## General FAQ

### Is Webots Really Free and Open-Source?

Yes!

Since the [R2019a version](../blog/Webots-2019-a-release.md), Webots is completely free and open-source.

> It remains in active development.

Reviewer: ```suggestion
It remains under active development.
```

> Sources are available on [GitHub](https://github.com/cyberbotics/webots) under [the Apache 2 license](https://github.com/cyberbotics/webots/blob/master/LICENSE).

Reviewer: ```suggestion
The source code is available from [GitHub](https://github.com/cyberbotics/webots) and released under the terms of the [Apache 2 license](https://github.com/cyberbotics/webots/blob/master/LICENSE).
```

> Cyberbotics takes part to many academic and industrial projects, and provides a paid user support, training and consulting for users who need to quickly develop high-quality Webots simulations.

Reviewer: ```suggestion
Cyberbotics collaborates to several academic and industrial projects, and provides a paid user support, training and consulting for users who need to quickly develop high-quality Webots simulations.
```

Please [contact us](mailto:info@cyberbotics.com) for more information about our services.

### Where to Start?

> [Download Webots](https://cyberbotics.com/download), [install it](installation-procedure.md) and perform [the official tutorial](tutorials.md).

Reviewer: ```suggestion
[Download Webots](https://cyberbotics.com/download), [install it](installation-procedure.md) and follow the [official tutorials](tutorials.md).
```

### Where Can I Find Help?

Join [our active community on Discord](https://discordapp.com/invite/nTWbN9m).

> The Cyberbotics team is available (at least) on european buisness hours.

Reviewer: ```suggestion
The Cyberbotics team is available (at least) during European business hours.
```

### Where Can I Ask a Question of General Interest?

On [StackOverflow with the `webots` tag](https://stackoverflow.com/questions/tagged/webots).

> The Cyberbotics team monitor this tag.

Reviewer: ```suggestion
The Cyberbotics team monitors this tag.
```

### How Can I Keep Informed?

Follow us on:

> - [our Newsletter](https://cyberbotics.com/news/subscribe.php)

Reviewer: ```suggestion
- the [Webots Newsletter](https://cyberbotics.com/news/subscribe.php)
```

- [Twitter](https://twitter.com/webots)
- [YouTube](http://www.youtube.com/user/cyberboticswebots)
- [LinkedIn](https://www.linkedin.com/company/20132793)

### How Can I Report a Bug in Webots?

Report the bug on [GitHub issues of the Webots main repository](https://github.com/cyberbotics/webots/issues/new/choose).

> Please make sure the issue comes really from Webots and give us simple and precise instructions to reproduce the issue.

Reviewer: ```suggestion
Please make sure the issue comes really from Webots and give us simple and precise instructions to reproduce it.
```

### Webots Crashes at Startup, What Can I Do?

1. Verify that your system is above the [minimal requirements to run Webots](system-requirements.md).
2. Update [your GPU drivers](verifying-your-graphics-driver-installation.md). *Webots uses intensively your GPU.*
3. Update Webots. *Webots is regularly patched to support as much hardware as possible.*
4. Try the [safe mode](starting-webots.md#safe-mode).

> 5. Fill a [crash report](https://github.com/cyberbotics/webots/issues/new/choose).

Reviewer: ```suggestion
5. Fill in a [crash report](https://github.com/cyberbotics/webots/issues/new/choose).
```
